tests/js/mock-survey-endpoints.js

```diff
@@ -41,13 +37,8 @@ export const mockSurveyEndpoints = () => {
 		body: {},
 	} );
 
-	fetchMock.getOnce( surveyTimeoutEndpoint, {
```

Comment:
This mock was incorrect to start with since the `survey-timeout` endpoint was POST only – `survey-timeouts` has GET.